hack/build-and-push.sh

```diff
@@ -479,9 +530,15 @@ build_push_tasks() {
             task_bundle_with_digest=${task_bundle}@${digest}
             echo "info: use existing $task_bundle_with_digest" 1>&2
         else
+            echo "info: finding the previous task bundle that has a migration" 1>&2
+            previous_migration_bundle_digest=$(find_previous_migration_bundle_digest "$task_name" "$task_version")
+
             echo "info: push new bundle $task_bundle" 1>&2
 
-            output=$(build_push_task "$task_dir" "$prepared_task_file" "$task_bundle" "$task_file_sha" "$has_migration")
+            output=$(
+                build_push_task "$task_dir" "$prepared_task_file" "$task_bundle" "$task_file_sha" \
+                    "$has_migration" "$previous_migration_bundle_digest"
```

Comment:
Is `previous_migration_bundle_digest` used anywhere?

Reply:
It is used in `build_push_task` https://github.com/konflux-ci/build-definitions/pull/1861/files#diff-78a47706e6b9aca3579f93bc34adac628f9c102de7257a2fe14939130786a807R234.